src/mongocxx/client.cpp

```diff
@@ -28,16 +31,8 @@ client::client() noexcept = default;
 client::client(const class uri& uri, const options::client& options)
     : _impl(bsoncxx::stdx::make_unique<impl>(libmongoc::client_new_from_uri(uri._impl->uri_t))) {
```

Comment:
Fixed! Since I wasn't changing any lines with `bsoncxx::stdx`, wasn't sure whether or not to change.